src/App.tsx

```diff
@@ -1,20 +1,27 @@
 
-import './App.scss'
+import React from 'react';
+import './App.scss';
+import { Link, Outlet } from 'react-router-dom';
 
-function App() {
-  
 
+function App() {
   return (
     <>
-      
-      <h1>Vite + React</h1>
-      <div className="card">
-      
-        
-      </div>
-     
+      <header>
+        <nav>
+          <Link to="/">Home</Link>
```

Comment:
You can also save all the paths in a constants file and then  use that.
E.g. 
`const paths = {HOME: "/home", CASE: "/case" etc}`

and then import it and use `paths.CASE`

This would help to avoid typos, and reuse paths easier

Reply:
Created an additional file with paths.